tests/unit/elasticity/test_elastic.py

```diff
@@ -150,6 +150,7 @@ def test_proper_mbsz(ds_config):
 class TestNonElasticBatchParams(DistributedTest):
     world_size = 2
 
+    @pytest.mark.skipif(not deepspeed.ops.__compatible_ops__[FusedLambBuilder.NAME], reason="lamb is not compatible")
```

Comment:
@vshekhawat-hlab - could you run the pre-commit checks on this, that will fix the formatting check on this PR.

Reply:
Thanks!